_episodes/51.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: episode
+title: "51: Trino cools off with PopSQL"
+date: 2023-07-06
+tags: trino popsql query editor collaboration
+youtube_id: ""
+wistia_id: ""
+sections:
+- title: "Intro"
+  desc: "An intro to PopSQL"
+  time: 0
+---
+
+## Hosts
+
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Starburst](https://starburst.io)
+* Manfred Moser, Director of Technical Content at
+  [Starburst](https://starburst.io)
+  ([@simpligility](https://twitter.com/simpligility))
+
+## Guests
+
+* Jake Peterson, Head of Customer Success at [PopSQL](https://popsql.com/)
+* Matt Peveler, Software Engineer at [PopSQL](https://popsql.com/)
+
+## Releases 423-427
+
+To be filled out shortly...
+
+## Concept of the episode: PopSQL
+
+It may be familiar to some of our viewers to describe an environment where
+key queries and dashboards are buried in someone's personal workspace, and you
+have to go ask them directly every time you want to check on a tool's health.
+When you're running a world-class, highly-performant query engine like Trino and
+investing time and resources into maintaining it, shouldn't you treat the end
+product like a first-class, collaborative, versioned system, too?
+
+[PopSQL](https://popsql.com/), a playful spin on the word popsicle, solves the
+sadness that is disorganized and siloed insights by centralizing queries into a
+collaborative platform that has versioning, security, and a suite of
+collaborative tools comparable to Google Drive. Want to work with your teammate
+on a query? You can open up the same editor and see the same thing. Want to see
+what that query someone ran last week was to see how the new feature is doing?
+It's there. Have a suggestion to improve something? Leave a comment. Realize
+your suggestion was wrong and need to undo the change? You can view past
+versions of the query.
+
+PopSQL and Trino make sense together. PopSQL provides a best-in-class interface
+for organizing, collaborating, and working together on all of your SQL queries
+across the business, and Trino handles running those queries at unparalleled
+speeds. They go hand-in-hand for treating your data and SQL analytics as first
+class citizens. In today's episode, we'll be exploring what PopSQL is, how it
+integrates with Trino, and how the engineers at PopSQL have done some cool
+things with Trino to make the integration better than ever before. We'll start
+with that last one, actually.
+
+## Concept of the episode: A new Node.js adapter for Trino
+
+Trino in the frontend is... a tricky thing. We can go ahead and admit that the
+[Trino web UI](/docs/current/admin/web-interface.html) isn't going to win any
+awards for design or functionality. And while a couple Node-based clients exist
```

Comment:
those are not clients.. they are client libraries .. so you could make a client with them ... like the JDBC driver is not a client app either